test/testsuites/csctargetnamespacetests.go

```diff
@@ -1,65 +1,51 @@
-package e2e
+package testsuites
 
 import (
 	"fmt"
 	"testing"
 
 	operator "github.com/operator-framework/operator-marketplace/pkg/apis/operators/v1"
+	helpers "github.com/operator-framework/operator-marketplace/test/helpers"
 	"github.com/operator-framework/operator-sdk/pkg/test"
 	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/wait"
 )
 
-const (
-	// nonExistingTargetNamespaceCSCName is the name of the catalogsourceconfig that points
-	// to a non-existing targetNamespace
-	nonExistingTargetNamespaceCSCName string = "non-existing-namespace-operators"
-
-	// targetNamespace is the non-existing target namespace
-	targetNamespace string = "non-existing-namespace"
-)
+// CSCTargetNamespace is a test suit that confirms that the targetNamespace field within
```

Comment:
Please `Csc` instead of `CSC` everywhere as that is the pattern we have been following so far.

Reply:
Understood, will do